### PR TITLE
Enable the polls feature

### PR DESCRIPTION
**Pull request by andybalaam**
This change removes the Polls labs flag, making the Create Poll button visible in the composer, and making existing polls visible in the timeline.

It does not change the fact that polls events still use an unstable prefix.

---
Here's what your changelog entry will look like:

## ✨ Features
 * Enable the polls feature ([\#7581](https://github.com/matrix-org/matrix-react-sdk/pull/7581)). Contributed by @andybalaam.

Preview: https://61e81b7a1fe03b15d1cadf14--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by jakewb-b — APPROVED** [review]
All good - thanks!

File: src/components/views/rooms/EventTile.tsx [hunk 1]
```diff
@@ -178,13 +178,6 @@ export function getHandlerTile(ev: MatrixEvent): string {
         }
     }
 
-    if (
-        M_POLL_START.matches(type) &&
```
Comment (dbkr) [inline src/components/views/rooms/EventTile.tsx:182]: Was this part of the condition important?

**Review by dbkr — APPROVED** [review]
Other than the sanity check, looks good: 🚢

File: src/components/views/rooms/EventTile.tsx (hunk 1 above)
Comment (andybalaam) [reply src/components/views/rooms/EventTile.tsx:182]: No, it was just - "if this is a poll event, and we don't have the polls feature, bail out."

**Pull request merged**